spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -453,7 +453,7 @@ public void testRewriteFilterRemoveDangling() throws Exception {
             .execute();
 
     List<DeleteFile> newDeleteFiles = except(deleteFiles(table), deleteFiles);
-    assertThat(newDeleteFiles).as("Should have 2 new delete files").hasSize(0);
+    assertThat(newDeleteFiles).as("Should have no new delete files").hasSize(0);
```

Comment:
does isEmpty work for case where size = 0?

Reply:
Done.